### PR TITLE
Remove requirement for inverters to have successors

### LINKED ISSUE #64: Graph loading does not handle PV inverters
### What happened?

I tried to load the graph for the site that has a PV inverter (actually a sandbox) and it failed because there is no successor component.

Code:
```
38    service = "{}:{}".format(
39        config["network"]["microgrid_host"],
40        config["network"]["microgrid_port"]
41    )
42    grpc_channel = grpc.insecure_channel(service)
43
44    client = MicrogridGrpcClient(grpc_channel, service)
45    graph = _MicrogridComponentGraph()
46    await graph.refresh_from_api(client)
```

Error:
```
(sdk_venv) david@david-desktop:~/projects/frequenz/source/mos-actor-fcr$ PYTHONPATH=src python -m frequenz.app.fcr.__main__ config/sandbox.toml
Traceback (most recent call last):
  File "/home/david/projects/frequenz/source/sdk_venv/lib/python3.10/site-packages/frequenz/sdk/microgrid/graph.py", line 322, in refresh_from
    _provisional.validate()
  File "/home/david/projects/frequenz/source/sdk_venv/lib/python3.10/site-packages/frequenz/sdk/microgrid/graph.py", line 357, in validate
    self._validate_intermediary_components()
  File "/home/david/projects/frequenz/source/sdk_venv/lib/python3.10/site-packages/frequenz/sdk/microgrid/graph.py", line 501, in _validate_intermediary_components
    raise InvalidGraphError(
frequenz.sdk.microgrid.graph.InvalidGraphError: Intermediary components without graph successors: [Component(component_id=4001, category=<ComponentCategory.INVERTER: 3>)]

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/david/projects/frequenz/source/mos-actor-fcr/src/frequenz/app/fcr/__main__.py", line 77, in <module>
    main()
  File "/home/david/projects/frequenz/source/mos-actor-fcr/src/frequenz/app/fcr/__main__.py", line 73, in main
    asyncio.run(run(config))
  File "/usr/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/david/projects/frequenz/source/mos-actor-fcr/src/frequenz/app/fcr/__main__.py", line 46, in run
    await graph.refresh_from_api(client)
  File "/home/david/projects/frequenz/source/sdk_venv/lib/python3.10/site-packages/frequenz/sdk/microgrid/graph.py", line 350, in refresh_from_api
    self.refresh_from(set(components), set(connections), correct_errors)
  File "/home/david/projects/frequenz/source/sdk_venv/lib/python3.10/site-packages/frequenz/sdk/microgrid/graph.py", line 324, in refresh_from
    raise InvalidGraphError(
frequenz.sdk.microgrid.graph.InvalidGraphError: Cannot populate component graph from provided input!
```

### What did you expect instead?

Graph loading should not expect PV inverters to have successor components.

### Affected version(s)

_No response_

### Affected part(s)

Microgrid (API, component graph, etc.) (part:microgrid)

### Extra information

_No response_

### PR DESCRIPTION
**Pull request by shsms**
This is a quick fix to allow PV inverters to be present in the graph.

This will be partially reverted later by expecting battery inverters to still have graph successors.

### CODE REVIEW

**Review by leandro-lucarella-frequenz — DISMISSED** [review]
Without knowing much about the implications, LGTM. Just add a RELEASE_NOTES entry.

Maybe @ela-kotulska-frequenz should have a look too, but if you are confident about it feel free to merge after adding the note.

File: RELEASE_NOTES.md
Comment (leandro-lucarella-frequenz) [inline RELEASE_NOTES.md:22]: Maybe add an issue for this if there isn't one yet?

**Review by leandro-lucarella-frequenz — APPROVED** [review]

File: RELEASE_NOTES.md
Comment (shsms) [reply RELEASE_NOTES.md:22]: https://github.com/frequenz-floss/frequenz-sdk-python/issues/89

**Pull request merged**
